README.md

```diff
@@ -1,2 +1,43 @@
-# Tech-Documentation
-This is a dashboard page that displays a hypothetical technical documentation. Built with HTML CSS.
+# Portfolio
+This is a portfolio page that displays a about and project sections. Built with HTML CSS.
+
+## Built With ✔️
+
+- HTML 
+- CSS
+
+
+## Live Demo ✔️
+
+[Live Demo Link](https://kamba56.github.io/Portfolio/)
+
+## Getting Started 🙌
+
+To get a local copy up and running follow these simple example steps.
+
```

Comment:
Add steps to get a local copy up and running.

Reply:
It would be better if the comment is phrased in a polite manner as a suggestion instead of an order. 

> Example: I think it's better if you add steps to get a local copy up and running. This is helpful for non-technical users to get a guide on how to use your repository.